### PR TITLE
docs: add doc of "DataFrame" in "Sheet Management" of chapter "FEATURES"

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2bgyqzc SDK 文档编写 —— Features > Sheet Management > Sheet](https://app.clickup.com/t/2bgyqzc)

File: docs/source/features/sheet_management/dataframe.rst [hunk 1]
```diff
@@ -4,3 +4,123 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the columns Protex schema defines and the data resides in the rows and columns.
+
+*********************
+ Construct DataFrame
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:13]: `Initialize  a Dataframe`. The following descriptions all start with `Initialize`.

**Review by graczhual — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/dataframe.rst [hunk 2]
```diff
@@ -4,3 +4,123 @@
...
+
+****************
+ Edit DataFrame
+****************
+
+Modify Column Content:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"] = [1, 4, 9]
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:83]: do we need to divide it to `add columns` and `modify data`? If updating data is mentioned here, do we also need to mention that series changes affect dataframe, etc

File: docs/source/features/sheet_management/dataframe.rst [hunk 3]
```diff
@@ -4,3 +4,123 @@
...
+
+Extend rows to the end of the DataFrame:
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:85]: should we add the ref to the function here?

File: docs/source/features/sheet_management/dataframe.rst (hunk 2 above)
Comment (AChenQ) [reply docs/source/features/sheet_management/dataframe.rst:83]: We not support add columns and series changes affect dataframe now...

File: docs/source/features/sheet_management/dataframe.rst [hunk 4]
```diff
@@ -4,3 +4,125 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the columns Protex schema defines and the data resides in the rows and columns.
+
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:11]: Can we explain it more clearly? break it down into a few sentences

File: docs/source/features/sheet_management/dataframe.rst [hunk 5]
```diff
@@ -4,3 +4,125 @@
...
+************************
+ Initialize a DataFrame
+************************
+
+Initialize a Dataframe by a List:
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:16]: do we need to add the restrictions about the list? like the descriptions about DataFrame in pandas https://pandas.pydata.org/docs/user_guide/dsintro.html#dataframe

Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:16]: do we need to add the restrictions about the list? like the descriptions about DataFrame in pandas  https://pandas.pydata.org/docs/user_guide/dsintro.html#dataframe

File: docs/source/features/sheet_management/dataframe.rst [hunk 6]
```diff
@@ -4,3 +4,125 @@
...
+
+.. code:: python
+
+   from graviti import DataFrame
+   data = [
+       {"filename": "a.jpg"},
+       {"filename": "b.jpg"},
+       {"filename": "c.jpg"},
+   ]
+   df = DataFrame(data)
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:26]: should we add printed results to help users understand?

File: docs/source/features/sheet_management/dataframe.rst [hunk 7]
```diff
@@ -4,3 +4,138 @@
...
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...      {"attribute": {"weather": "sunny", "color": "red"}},
+   ...      {"attribute": {"weather": "rainy", "color": "black"}},
+   ...      {"attribute": {"weather": "sunny", "color": "white"}},
+   ...  ]
+   >>> df = DataFrame(data)
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:42]: `DataFrame(data)` or add `>>> df`. same problems in other examples

**Review by graczhual — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/dataframe.rst [hunk 8]
```diff
@@ -4,3 +4,141 @@
...
+ Edit DataFrame
+****************
+
+Modify Column Content:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"] = [1, 4, 9]
+
+Graviti SDK support use :py:meth:`~graviti.dataframe.frame.DataFrame.extend` method to extend DataFrame.
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:101]: ```DataFrame supports method :py:meth:`~graviti.dataframe.frame.DataFrame.extend`:```

**Review by graczhual — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/dataframe.rst [hunk 9]
```diff
@@ -4,3 +4,141 @@
...
+Initialize a DataFrame with Multi-level column names:
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...      {"attribute": {"weather": "sunny", "color": "red"}},
+   ...      {"attribute": {"weather": "rainy", "color": "black"}},
+   ...      {"attribute": {"weather": "sunny", "color": "white"}},
+   ...  ]
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:42]: There seems to be some problems with indentation, please check all examples.

File: docs/source/features/sheet_management/dataframe.rst [hunk 10]
```diff
@@ -4,3 +4,141 @@
...
+   ...    {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}]},
+   ...    {"points": [{"xmin": 6, "ymin": 10}]},
+   ...    {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}, {"xmin": 1, "ymin": 9}]},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      points
+   0  DataFrame(2, 2)
+   1  DataFrame(1, 2)
+   2  DataFrame(3, 2)
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:66]: add the printed result of df["points"][0]

**Review by graczhual — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/dataframe.rst [hunk 11]
```diff
@@ -4,3 +4,145 @@
...
+
+****************
+ Edit DataFrame
+****************
+
+Modify Column Content:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"] = [1, 4, 9]
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:103]: Now we only support changing the column content of local DataFrame. do not need to add this method now.

File: docs/source/features/sheet_management/dataframe.rst [hunk 12]
```diff
@@ -4,3 +4,145 @@
...
+   0  DataFrame(2, 2)
+   1  DataFrame(1, 2)
+   2  DataFrame(3, 2)
+   >>> df["points"][0]
+      xmin  ymin
+   0  1     3
+   1  5     8
+
+****************
+ Read DataFrame
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:73]: Read a Dataframe

File: docs/source/features/sheet_management/dataframe.rst [hunk 13]
```diff
@@ -4,3 +4,145 @@
...
+
+Read a DataFrame cell:
+
+.. code:: python
+
+   df.loc[0][f"{COLUMN_NAME}"]
+   df[f"{COLUMN_NAME}"][0]
+
+****************
+ Edit DataFrame
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:96]: a

File: docs/source/features/sheet_management/dataframe.rst [hunk 14]
```diff
@@ -4,3 +4,139 @@
...
+
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   schema = pt.record({"file": ob.file.RemoteFile()})
+   data = [
+       {"file": File("PATH/TO/YOUR/FILE")},
+       {"file": File("b.jpg")},
+   ]
+   df = DataFrame(data, schema)
+
+Read the contents of :class:`~graviti.utility.file.File` and :class:`~graviti.utility.file.RemoteFile`:
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:137]: the class RemoteFile also represents the file in df? Can we just say `Read the file in the DataFrame.`?

File: docs/source/features/sheet_management/dataframe.rst [hunk 15]
```diff
@@ -4,3 +4,139 @@
...
+
+DataFrame supports method :py:meth:`~graviti.dataframe.frame.DataFrame.extend`:
+
+Extend rows to the end of the DataFrame:
+
+.. code:: python
+
+   df.extend([{"filename": "a.jpg"}])
+
+Extend other Dataframe to the end of the DataFrame:
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:107]: another

File: docs/source/features/sheet_management/dataframe.rst [hunk 16]
```diff
@@ -4,3 +4,139 @@
...
+
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   schema = pt.record({"file": ob.file.RemoteFile()})
+   data = [
+       {"file": File("PATH/TO/YOUR/FILE")},
+       {"file": File("b.jpg")},
+   ]
+   df = DataFrame(data, schema)
+
+Read the file in :class:`~graviti.utility.file.File` and :class:`~graviti.utility.file.RemoteFile`:
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:137]: what is the file in class File and Remotefile

**Review by graczhual — APPROVED** [review]

File: docs/source/features/sheet_management/dataframe.rst [hunk 17]
```diff
@@ -4,3 +4,139 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the 2-dimensional data with columns of Protex schema.
```
Comment (Lee-000) [inline docs/source/features/sheet_management/dataframe.rst:10]: `A Graviti DataFrame contains 2-dimensional tabular data and a Protex schema describing the names and types of each column.` 
Would this be more clear?

File: docs/source/features/sheet_management/dataframe.rst [hunk 18]
```diff
@@ -4,3 +4,139 @@
...
+   ...    {"filename": "c.jpg"},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      filename
+   0  a.jpg
+   1  b.jpg
+   2  c.jpg
+
+Initialize a DataFrame with Multi-level column names:
```
Comment (Lee-000) [inline docs/source/features/sheet_management/dataframe.rst:33]: `multi-level`?

File: docs/source/features/sheet_management/dataframe.rst [hunk 19]
```diff
@@ -4,3 +4,139 @@
...
+
+Load the local file into DataFrame:
+
+.. code:: python
+
+   import graviti.portex as pt
+   from graviti import DataFrame
+   from graviti.utility import File
+
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
```
Comment (Lee-000) [inline docs/source/features/sheet_management/dataframe.rst:129]: Do we use `ob` or `standard`?

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**